### ISSUE TITLE
Documentation and Print Query

### ISSUE THREAD

**Issue by tiagodavi** [orientechnologies/orientjs#222]
Hi guys. 

How can I print a specific query on terminal? and Where can I find new documentation? I saw that documentation is older than code..

**Comment by maggiolo00** [comment]
Hi @tiagodavi 

did you look at this?

http://orientdb.com/docs/last/OrientJS-Query.html

**Comment by tiagodavi** [comment]
Yes, thank you. But I saw that your tests have functionalities that are not documented yet.
